pkg/cmd/util/clientcmd/factory.go

```diff
@@ -201,7 +201,11 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if !ok {
 				return nil, errors.New("provided options object is not a BuildLogOptions")
 			}
-			buildsForBCSelector := labels.SelectorFromSet(map[string]string{buildapi.DeprecatedBuildConfigLabel: t.Name})
+			buildConfigLabels := labels.Set{
+				buildapi.BuildConfigLabel: t.Name,
```

Comment:
because we want to at least stop setting the old value on new objects.

and someday perhaps we'll handle migrations cleanly enough that we can in fact purge/fix the old values.